### DISCUSSION TITLE
Disable Enhance Pointer Precision in Mouse Options

### DISCUSSION THREAD

**Discussion by Zoltan000** [farag2/Sophia-Script-for-Windows#553] [Ideas]
would be nice to have the script disable the 'enhance pointer precision' option
![image_2024-02-10_232204198](https://github.com/farag2/Sophia-Script-for-Windows/assets/89108321/1e0086c4-3ddd-48b4-8e3a-60b64e4c1b27)

**Comment by farag2** [comment]
I do not consider adding such things yet. For what?)

https://www.elevenforum.com/t/turn-on-or-off-enhance-pointer-precision-in-windows-11.7327/#Two

**Reply by yuzamei2854** [reply]
Probably for the consistency of cursor movement, as this acceleration can be bad in some games.

**Comment by soredake** [comment]
I personally hate mouse acceleration.
